x-pack/plugins/alerts/server/invalidate_pending_api_keys/add_to_invalidate_api_keys.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Logger, ISavedObjectsRepository } from 'src/core/server';
+
+export const addToInvalidateApiKeys = async (
```

Comment:
**optional nit:** I wonder if the terminology would be clearer if it was `markApiKeyForInvalidation` or something along those lines?